### PR TITLE
Updating documentation on cli.js location and 3rd party repo links

### PR DESCRIPTION
**Pull request by Neyromancer**
## Description
Some part of ducmentation is outdated which makes it difficult to follow it.

## What was done?
1.  dist/cli.js location updated, from `dist/cli.js` to cli/dist/cli.js. Changes introduced with [PR](https://github.com/paritytech/zombienet/pull/475/files).
2. links to polkadot repo replaced with links to polkadot-sdk repo as polkadot repo was archived. [See](https://github.com/paritytech/polkadot/blob/52209dcfe546ff39cc031b92d64e787e7e8264d4/README.md?plain=1#L3)
3.  Removed one Note as excessive.

Relates to [Issue #1412](https://github.com/paritytech/zombienet/issues/1412) Error when trying to spawn network in kubernetes

------------------------------
Tasks:
- [ ] Code review
- [ ] Validate script's changes by running them locally

### CODE REVIEW

File: javascript/package.json [hunk 1]
```diff
@@ -21,15 +21,15 @@
     "package": "npm run -w packages/cli package",
     "package:linux": "npm run -w packages/cli package:linux",
     "package:macos": "npm run -w packages/cli package:macos",
-    "zombie": "node ./packages/cli/dist/cli.js",
+    "zombie": "node ./packages/cli/src/cli.ts",
```
Comment (wirednkod) [inline javascript/package.json:24]: This change does not feel correct

File: scripts/ci/docker/zombienet_injected.Dockerfile [hunk 2]
```diff
@@ -47,8 +47,8 @@ RUN npm install --production
 RUN chown -R nonroot. /home/nonroot
 
 # Change `cli` permissions and link to easy call
-RUN chmod +x /home/nonroot/zombie-net/packages/cli/dist/cli.js
-RUN ln -s /home/nonroot/zombie-net/packages/cli/dist/cli.js /usr/local/bin/zombie
+RUN chmod +x /home/nonroot/zombie-net/packages/cli/src/cli.ts
```
Comment (wirednkod) [inline scripts/ci/docker/zombienet_injected.Dockerfile:50]: Same as above.

**Review by wirednkod — CHANGES_REQUESTED** [review]

File: javascript/package.json (hunk 1 above)
Comment (Neyromancer) [reply javascript/package.json:24]: yeah, committed it by coincidence.

File: scripts/ci/docker/zombienet_injected.Dockerfile (hunk 2 above)
Comment (Neyromancer) [reply scripts/ci/docker/zombienet_injected.Dockerfile:50]: commited by coincidence

File: README.md [hunk 3]
```diff
@@ -331,7 +331,8 @@ npm run build
 For an easier and faster setup of your local environment, run:
 
 ```bash
-node dist/cli.js setup <binaries>
+cd ./<zombinet project dir>/javascript/packages
+node ./cli/dist/cli.js <binaries>
```
[the remark this answers is not in the document]
Comment (Neyromancer) [reply README.md:335]: I have the following directory tree structure
```
> javascript % tree -L 3 -I node_modules -d

.
└── packages
   ├── cli
   │   ├── dist
   │   └── src
   ├── orchestrator
   │   ├── dist
   │   ├── src
   │   └── static-configs
   └── utils
        ├── dist
        └── src
```
where `cli.js` is in `cli/dist/cli.js`.

When i am trying to execute `node dist/cli.js version` from `javascript` or `javascript/packages` i've got error message `Error: Cannot find module '/Volumes/KINGSTON/opensource_projects/zombienet/javascript/dist/cli.js`. If i try to call `node cli/dist/cli.js version` from `javasript` dir i get the same error. However if to call `node cli/dist/cli.js version` from `javascript/packages` i get the current cli version.

[the remark this answers is not in the document]
Comment (Neyromancer) [reply README.md:335]: @wirednkod 

as far as i understand cli.ts was [moved](https://github.com/paritytech/zombienet/blob/3a9a10610592bcf380a4007a7d4517b3e33769bc/javascript/packages/cli/src/cli.ts) from src/cli.ts to packages/cli/cli.ts with this [PR](https://github.com/paritytech/zombienet/pull/475/files). It seems for me that we need to update the previosly working call node `dist/cli.js` with `node cli/dist/cli.js`. And we have to do this from `packages` directory as currently `cli/dist/cli.js` is located in `packages`

File: README.md [hunk 4]
```diff
@@ -331,7 +331,8 @@ npm run build
 For an easier and faster setup of your local environment, run:
 
 ```bash
-node dist/cli.js setup <binaries>
+❯ cd <zombinet project dir>/javascript/packages
```
Comment (Neyromancer) [inline README.md:334]: I have the following directory tree structure
```
> javascript % tree -L 3 -I node_modules -d

.
└── packages
   ├── cli
   │   ├── dist
   │   └── src
   ├── orchestrator
   │   ├── dist
   │   ├── src
   │   └── static-configs
   └── utils
        ├── dist
        └── src
```
where `cli.js` is in `cli/dist/cli.js`.

When i am trying to execute `node dist/cli.js version` from `javascript` or `javascript/packages` i've got error message `Error: Cannot find module '/zombienet/javascript/dist/cli.js`. If i try to call `node cli/dist/cli.js version` from `javasript` dir i get the same error. However if to call `node cli/dist/cli.js version` from `javascript/packages` i get the current cli version.

Comment (Neyromancer) [reply README.md:334]: as far as i understood `cli.ts` was [moved](https://github.com/paritytech/zombienet/blob/3a9a10610592bcf380a4007a7d4517b3e33769bc/javascript/packages/cli/src/cli.ts) from src/cli.ts to packages/cli/cli.ts with this [PR](https://github.com/paritytech/zombienet/pull/475/files). It seems for me that we need to update the previosly working call node `dist/cli.js` with `node cli/dist/cli.js`. And we have to do this from `packages` directory as currently `cli/dist/cli.js` is located in `packages`

Comment (Neyromancer) [reply README.md:334]: this comment relates to all similar changes around the project

File: README.md [hunk 5]
```diff
@@ -28,9 +28,6 @@ Internally zombienet is a `javascript` library, designed to run on `Node.js` and
 backend `providers` to run the *nodes*, at this moment `kubernetes`, `podman` and `native` are
 supported.
 
-**Note:** Currently, it is only possible to use `podman` for Zombienet users on Linux machines.
```
Comment (Neyromancer) [inline README.md:31]: This `Note` totally repeats `Note` [here](https://github.com/paritytech/zombienet/blob/a7a287f8237c2dcce1c3c65a52b4e1130a344461/README.md?plain=1#L41)

File: README.md [hunk 6]
```diff
@@ -120,7 +117,7 @@ Native provider doesn't run any extra layer/process at the moment.
 *For this example we will use the `macos` version of the executable*
 
 ```bash
-./zombienet-macos
+❯ ./zombienet-macos
```
Comment (Neyromancer) [inline README.md:120]: to make all bash snippets alike

applicable to all changes around the project's docs

File: README.md [hunk 7]
```diff
@@ -343,27 +341,28 @@ You can use the following arguments:
 For example:
 
 ```bash
-node dist/cli.js setup polkadot polkadot-parachain
+❯ cd zombinet/javascript/packages
```
Comment (Neyromancer) [inline README.md:344]: I have the following directory tree structure
```
> javascript % tree -L 3 -I node_modules -d

.
└── packages
   ├── cli
   │   ├── dist
   │   └── src
   ├── orchestrator
   │   ├── dist
   │   ├── src
   │   └── static-configs
   └── utils
        ├── dist
        └── src
```
where `cli.js` is in `cli/dist/cli.js`.

When i am trying to execute `node dist/cli.js version` from `javascript` or `javascript/packages` i've got error message `Error: Cannot find module '/zombienet/javascript/dist/cli.js`. If i try to call `node cli/dist/cli.js version` from `javasript` dir i get the same error. However if to call `node cli/dist/cli.js version` from `javascript/packages` i get the current cli version.

Comment (Neyromancer) [reply README.md:344]: as far as i understood `cli.ts` was [moved](https://github.com/paritytech/zombienet/blob/3a9a10610592bcf380a4007a7d4517b3e33769bc/javascript/packages/cli/src/cli.ts) from src/cli.ts to packages/cli/cli.ts with this [PR](https://github.com/paritytech/zombienet/pull/475/files). 

It seems for me that we need to update the previosly working call node `dist/cli.js` with `node cli/dist/cli.js`. And we have to do this from `packages` directory as currently `cli/dist/cli.js` is located in `packages`

File: README.md [hunk 8]
```diff
@@ -343,27 +341,28 @@ You can use the following arguments:
...
+❯ node ./cli/dist/cli.js setup polkadot polkadot-parachain
 ```
 
-> Note: If you are using macOS please clone the [Polkadot repo](https://github.com/paritytech/polkadot) and run it locally. At the moment there is no `polkadot` binary for MacOs.
+> Note: If you are using macOS please clone the [polkadot-sdk repo](https://github.com/paritytech/polkadot-sdk) and run it locally. At the moment there is no `polkadot` binary for MacOs.
```
Comment (Neyromancer) [inline README.md:348]: polkadot repo archived and project moved to polkadot-sdk repo, [see](https://github.com/paritytech/polkadot/blob/52209dcfe546ff39cc031b92d64e787e7e8264d4/README.md?plain=1#L3)

File: README.md [hunk 9]
```diff
@@ -343,27 +341,28 @@ You can use the following arguments:
...
 
 The command above will retrieve the binaries provided and try to download and prepare those binaries for usage. 
 At the end of the download, the `setup` script will provide a command to run in your local environment in order to add the directory where the binaries were downloaded in your $PATH var, for example:
 
 ```bash
-Please add the dir to your $PATH by running the command: export PATH=/home/<user>/zombienet/dist:$PATH
+Please add the dir to your $PATH by running the command: export PATH=/<full path to the project>/zombienet/dist:$PATH
```
Comment (Neyromancer) [inline README.md:354]: It is possible that repo was cloned in a directory different from `/home/<user>`

File: README.md [hunk 10]
```diff
@@ -343,27 +341,28 @@ You can use the following arguments:
...
 ### Build adder-collator (needed for running examples with native provider)
 
 You can build it from source like this
 
 ```bash
-git clone git@github.com:paritytech/polkadot
-cd polkadot
-cargo build --profile testnet -p test-parachain-adder-collator
-export PATH=$(pwd)/target/testnet:$PATH
+❯ git clone git@github.com:paritytech/polkadot-sdk.git
```
Comment (Neyromancer) [inline README.md:362]: polkadot repo archived and project moved to polkadot-sdk repo, [see](https://github.com/paritytech/polkadot/blob/52209dcfe546ff39cc031b92d64e787e7e8264d4/README.md?plain=1#L3)

applicable to all similar changes around this PR

File: README.md [hunk 11]
```diff
@@ -343,7 +344,8 @@ You can use the following arguments:
 For example:
 
 ```bash
-node dist/cli.js setup polkadot polkadot-parachain
+cd cd ./<zombinet project dir>/javascript/packages
+node ./cli/dist/cli.js setup polkadot polkadot-parachain
```
[the remark this answers is not in the document]
Comment (Neyromancer) [reply README.md]: added comments about the changes

File: README.md (hunk 5 above)
Comment (Neyromancer) [reply README.md:31]: applicable to all similar changes

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (Neyromancer) [reply README.md]: @wirednkod, i think i understood what you meant. You might meant this example, right?
```
> node ./packages/cli/dist/cli.js spawn ../tests/chaos/0001-delay.toml 
```

These 2 ways of calling `cli.js`
```
>cd ./javascript/
> node ./packages/cli/dist/cli.js spawn ../tests/chaos/0001-delay.toml 
```
and
```
>cd ./javascript/packages
> node /cli/dist/cli.js spawn ../tests/chaos/0001-delay.toml 
```
look similar as for me however what one is better to use around the docs?

File: README.md (hunk 7 above)
Comment (wirednkod) [reply README.md:344]: I think there is a misunderstanding here. Each package has its own structure and when build creates its own `dist` directory. All `TS` is transpiled to `JS` insde that `dist` dir.
Having said that, with the structure you mentioned above, you will need indeed to run the command you mention... but first you need to run the build command. INSTEAD, though you can run the commands inside the package.json of `/zombienet/javascript` instead. e.g.:
`❯ npm run zombie -- spawn ....`
etc..etc..

File: README.md (hunk 9 above)
Comment (wirednkod) [reply README.md:354]: I really think that this is not needed to be so specific... but sure..

File: docs/book/development.html [hunk 12]
```diff
@@ -155,31 +155,31 @@ <h2 id="requirements"><a class="header" href="#requirements">Requirements</a></h
...
-npm install
-npm run build
+<pre><code class="language-bash">❯ cd zombienet
+❯ npm install
+❯ npm run build
 </code></pre>
 <h3 id="download-and-install-needed-artifacts-optional"><a class="header" href="#download-and-install-needed-artifacts-optional">Download and install needed artifacts (optional)</a></h3>
 <p>For an easier and faster setup of your local environment, run:</p>
-<pre><code class="language-bash">node dist/cli.js setup &lt;binaries&gt;
+<pre><code class="language-bash">❯ cd zombinet/javascript/packages && node ./cli/dist/cli.js setup &lt;binaries&gt;
```
Comment (wirednkod) [inline docs/book/development.html:164]:
```suggestion
<pre><code class="language-bash">❯ cd zombinet/javascript && npm run build zombie -- setup &lt;binaries&gt;
```

File: docs/book/development.html [hunk 13]
```diff
@@ -155,31 +155,31 @@ <h2 id="requirements"><a class="header" href="#requirements">Requirements</a></h
...
 </code></pre>
 <p>This allows to use the <code>setup</code> script, making everything ready for a ZombieNet dev environment.</p>
 <p>You can use the following arguments:</p>
 <p><code>--help</code> shows the different options and commands for using the Zombienet CLI.
 <code>--binaries</code> or <code>-b</code>: enables providing the binaries that you want to be downloaded and installed during the setup. Possible options: <code>polkadot</code>, <code>polkadot-parachain</code>.</p>
 <p>For example:</p>
-<pre><code class="language-bash">node dist/cli.js setup polkadot polkadot-parachain
+<pre><code class="language-bash">❯ cd zombinet/javascript/packages && node ./cli/dist/cli.js setup polkadot polkadot-parachain
```
Comment (wirednkod) [inline docs/book/development.html:171]:
```suggestion
<pre><code class="language-bash">❯ cd zombinet/javascript && npm run build zombie -- setup polkadot polkadot-parachain
```

File: docs/book/development.html [hunk 14]
```diff
@@ -155,31 +155,31 @@ <h2 id="requirements"><a class="header" href="#requirements">Requirements</a></h
...
+<p>Note: If you are using macOS please clone the <a href="https://github.com/paritytech/polkadot-sdk">polkadot-sdk repo</a> and run it locally. At the moment there is no <code>polkadot</code> binary for MacOs.</p>
 </blockquote>
 <p>The command above will retrieve the binaries provided and try to download and prepare those binaries for usage. 
 At the end of the download, the <code>setup</code> script will provide a command to run in your local environment in order to add the directory where the binaries were downloaded in your $PATH var, for example:</p>
 <pre><code class="language-bash">Please add the dir to your $PATH by running the command: export PATH=/home/&lt;user&gt;/current_directory:$PATH
 </code></pre>
 <h3 id="using-zombienet"><a class="header" href="#using-zombienet">Using Zombienet</a></h3>
 <p>With the above steps completed, the <code>zombienet</code> CLI is ready to run:</p>
-<pre><code class="language-bash">❯ node dist/cli.js
+<pre><code class="language-bash">❯ cd zombinet/javascript/packages && node ./cli/dist/cli.js
```
Comment (wirednkod) [inline docs/book/development.html:182]:
```suggestion
<pre><code class="language-bash">❯ cd zombinet/javascript && npm run zombie
```

File: docs/book/print.html [hunk 15]
```diff
@@ -1690,7 +1690,7 @@ <h3 id="download-and-install-needed-artifacts-optional">
             <p>
               For an easier and faster setup of your local environment, run:
             </p>
-            <pre><code class="language-bash">node dist/cli.js setup &lt;binaries&gt;
+            <pre><code class="language-bash">cd zombinet/javascript/packages && node ./cli/dist/cli.js setup &lt;binaries&gt;
```
Comment (wirednkod) [inline docs/book/print.html:1693]:
```suggestion
            <pre><code class="language-bash">cd zombinet/javascript && npm run zombie -- setup &lt;binaries&gt;
```

File: docs/book/print.html [hunk 16]
```diff
@@ -1705,13 +1705,13 @@ <h3 id="download-and-install-needed-artifacts-optional">
               <code>polkadot</code>, <code>polkadot-parachain</code>.
             </p>
             <p>For example:</p>
-            <pre><code class="language-bash">node dist/cli.js setup polkadot polkadot-parachain
+            <pre><code class="language-bash">cd zombinet/javascript/packages && node ./cli/dist/cli.js setup polkadot polkadot-parachain
```
Comment (wirednkod) [inline docs/book/print.html:1708]:
```suggestion
            <pre><code class="language-bash">cd zombinet/javascript && npm run zombie -- setup polkadot polkadot-parachain
```

File: docs/book/print.html [hunk 17]
```diff
@@ -1733,7 +1733,7 @@ <h3 id="using-zombienet">
               With the above steps completed, the <code>zombienet</code> CLI is
               ready to run:
             </p>
-            <pre><code class="language-bash">❯ node dist/cli.js
+            <pre><code class="language-bash"> cd zombinet/javascript/packages && node ./cli/dist/cli.js
```
Comment (wirednkod) [inline docs/book/print.html:1736]:
```suggestion
            <pre><code class="language-bash"> cd zombinet/javascript && npm run zombie --
```

File: docs/book/searchindex.js [hunk 18]
```diff
@@ -280,14 +280,14 @@ Object.assign(window.search, {
...
+          body: "For an easier and faster setup of your local environment, run: cd zombinet/javascript/packages && node ./cli/dist/cli.js setup <binaries> This allows to use the setup script, making everything ready for a ZombieNet dev environment. You can use the following arguments: --help shows the different options and commands for using the Zombienet CLI. --binaries or -b: enables providing the binaries that you want to be downloaded and installed during the setup. Possible options: polkadot, polkadot-parachain. For example: cd zombinet/javascript/packages && node ./cli/dist/cli.js setup polkadot polkadot-parachain Note: If you are using macOS please clone the polkadot-sdk repo and run it locally. At the moment there is no polkadot binary for MacOs. The command above will retrieve the binaries provided and try to download and prepare those binaries for usage. At the end of the download, the setup script will provide a command to run in your local environment in order to add the directory where the binaries were downloaded in your $PATH var, for example: Please add the dir to your $PATH by running the command: export PATH=/home/<user>/current_directory:$PATH",
```
Comment (wirednkod) [inline docs/book/searchindex.js:283]:
```suggestion
          body: "For an easier and faster setup of your local environment, run: cd zombinet/javascript && npm run zombie -- setup <binaries> This allows to use the setup script, making everything ready for a ZombieNet dev environment. You can use the following arguments: --help shows the different options and commands for using the Zombienet CLI. --binaries or -b: enables providing the binaries that you want to be downloaded and installed during the setup. Possible options: polkadot, polkadot-parachain. For example: cd zombinet/javascript && npm run zombie -- setup polkadot polkadot-parachain Note: If you are using macOS please clone the polkadot-sdk repo and run it locally. At the moment there is no polkadot binary for MacOs. The command above will retrieve the binaries provided and try to download and prepare those binaries for usage. At the end of the download, the setup script will provide a command to run in your local environment in order to add the directory where the binaries were downloaded in your $PATH var, for example: Please add the dir to your $PATH by running the command: export PATH=/home/<user>/current_directory:$PATH",
```

File: docs/book/searchindex.js [hunk 19]
```diff
@@ -280,14 +280,14 @@ Object.assign(window.search, {
...
           id: "37",
           title: "Download and install needed artifacts (optional)",
         },
         38: {
-          body: 'With the above steps completed, the zombienet CLI is ready to run: ❯ node dist/cli.js Usage: zombienet [options] [command] Options: -p, --provider <provider> Override provider to use (choices: "podman", "kubernetes", default: kubernetes) -h, --help display help for command Commands: spawn <networkConfig> [creds] [monitor] Spawn the network defined in the config test <testFile> Run tests on the network defined version Prints zombienet version help [command] display help for command',
+          body: 'With the above steps completed, the zombienet CLI is ready to run: cd zombinet/javascript/packages && node ./cli/dist/cli.js Usage: zombienet [options] [command] Options: -p, --provider <provider> Override provider to use (choices: "podman", "kubernetes", default: kubernetes) -h, --help display help for command Commands: spawn <networkConfig> [creds] [monitor] Spawn the network defined in the config test <testFile> Run tests on the network defined version Prints zombienet version help [command] display help for command',
```
Comment (wirednkod) [inline docs/book/searchindex.js:290]:
```suggestion
          body: 'With the above steps completed, the zombienet CLI is ready to run: cd zombinet/javascript && npm run zombie -- Usage: zombienet [options] [command] Options: -p, --provider <provider> Override provider to use (choices: "podman", "kubernetes", default: kubernetes) -h, --help display help for command Commands: spawn <networkConfig> [creds] [monitor] Spawn the network defined in the config test <testFile> Run tests on the network defined version Prints zombienet version help [command] display help for command',
```

File: docs/src/development.md [hunk 20]
```diff
@@ -12,17 +12,18 @@
...
 ```
 
 ### Download and install needed artifacts (optional)
 
 For an easier and faster setup of your local environment, run:
 
 ```bash
-node dist/cli.js setup <binaries>
+❯ cd zombinet/javascript/packages
+❯ node ./cli/dist/cli.js <binaries>
```
Comment (wirednkod) [inline docs/src/development.md:26]:
```suggestion
❯ cd zombinet/javascript
❯ npm run zombie -- setup <binaries>
```

File: docs/src/development.md [hunk 21]
```diff
@@ -35,10 +36,11 @@ You can use the following arguments:
 For example:
 
 ```bash
-node dist/cli.js setup polkadot polkadot-parachain
+❯ cd zombinet/javascript/packages
+❯ node ./cli/dist/cli.js setup polkadot polkadot-parachain
```
Comment (wirednkod) [inline docs/src/development.md:40]:
```suggestion
❯ cd zombinet/javascript
❯ npm run zombie --  setup polkadot polkadot-parachain
```

File: docs/src/development.md [hunk 22]
```diff
@@ -62,7 +64,8 @@ nix run nixpkgs#prefetch-npm-deps -- javascript/package-lock.json 2>/dev/null
 With the above steps completed, the `zombienet` CLI is ready to run:
 
 ```bash
-❯ node dist/cli.js
+❯ cd zombinet/javascript/packages
+❯ node ./cli/dist/cli.js
```
Comment (wirednkod) [inline docs/src/development.md:68]:
```suggestion
❯ cd zombinet/javascript/
❯ npm run zombie 
```

File: javascript/README.md [hunk 23]
```diff
@@ -317,17 +314,18 @@ alice: parachain 100 block height is at least 10 within 200 seconds
...
 ```
 
 ### Download and install needed artifacts (optional)
 
 For an easier and faster setup of your local environment, run:
 
 ```bash
-node dist/cli.js setup <binaries>
+❯ cd zombinet/javascript/packages
+❯ node ./cli/dist/cli.js setup <binaries>
```
Comment (wirednkod) [inline javascript/README.md:328]:
```suggestion
❯ cd zombinet/javascript
❯ npm run zombie --<binaries>
```

File: javascript/README.md [hunk 24]
```diff
@@ -340,10 +338,11 @@ You can use the following arguments:
 For example:
 
 ```bash
-node dist/cli.js setup polkadot polkadot-parachain
+❯ cd zombinet/javascript/packages
+❯ node ./cli/dist/cli.js setup polkadot polkadot-parachain
```
Comment (wirednkod) [inline javascript/README.md:342]:
```suggestion
❯ cd zombinet/javascript
❯ npm run zombie -- setup polkadot polkadot-parachain
```

File: javascript/README.md [hunk 25]
```diff
@@ -369,7 +368,8 @@ export PATH=$(pwd)/target/testnet:$PATH
 With the above steps completed, the `zombienet` CLI is ready to run:
 
 ```bash
-❯ node dist/cli.js
+❯ cd zombinet/javascript/packages
+❯ node ./cli/dist/cli.js
```
Comment (wirednkod) [inline javascript/README.md:372]:
```suggestion
❯ cd zombinet/javascript
❯ npm run zombie
```

File: javascript/packages/cli/README.md [hunk 26]
```diff
@@ -318,17 +315,18 @@ alice: parachain 100 block height is at least 10 within 200 seconds
...
 ```
 
 ### Download and install needed artifacts (optional)
 
 For an easier and faster setup of your local environment, run:
 
 ```bash
-node dist/cli.js setup <binaries>
+❯ cd zombinet/javascript/packages
+❯ node ./cli/dist/cli.js setup <binaries>
```
Comment (wirednkod) [inline javascript/packages/cli/README.md:329]:
```suggestion
❯ cd zombinet/javascript
❯ npm run zombie -- setup <binaries>
```

File: javascript/packages/cli/README.md [hunk 27]
```diff
@@ -370,7 +369,8 @@ export PATH=$(pwd)/target/testnet:$PATH
 With the above steps completed, the `zombienet` CLI is ready to run:
 
 ```bash
-❯ node dist/cli.js
+❯ cd zombinet/javascript/packages
+❯ node ./cli/dist/cli.js
```
Comment (wirednkod) [inline javascript/packages/cli/README.md:373]:
```suggestion
❯ cd zombinet/javascript
❯ npm run zombie 
```

File: javascript/packages/orchestrator/README.md [hunk 28]
```diff
@@ -329,24 +326,25 @@ Building the `parser-wrapper` can help on running/testing locally changes on the
...
 ```
 
 ### Download and install needed artifacts (optional)
 
 For an easier and faster setup of your local environment, run:
 
 ```bash
-node dist/cli.js setup <binaries>
+❯ cd zombinet/javascript/packages
+❯ node ./cli/dist/cli.js setup <binaries>
```
Comment (wirednkod) [inline javascript/packages/orchestrator/README.md:347]:
```suggestion
❯ cd zombinet/javascript
❯ npm run zombie -- setup <binaries>
```

File: javascript/packages/orchestrator/README.md [hunk 29]
```diff
@@ -359,10 +357,11 @@ You can use the following arguments:
 For example:
 
 ```bash
-node dist/cli.js setup polkadot polkadot-parachain
+❯ cd zombinet/javascript/packages
+❯ node ./cli/dist/cli.js setup polkadot polkadot-parachain
```
Comment (wirednkod) [inline javascript/packages/orchestrator/README.md:361]: ❯ cd zombinet/javascript
❯ npm run zombie -- setup polkadot  polkadot-parachain

File: javascript/packages/orchestrator/README.md [hunk 30]
```diff
@@ -388,7 +387,8 @@ export PATH=$(pwd)/target/testnet:$PATH
 With the above steps completed, the `zombienet` CLI is ready to run:
 
 ```bash
-❯ node dist/cli.js
+❯ cd zombinet/javascript/packages
+❯ node ./cli/dist/cli.js
```
Comment (wirednkod) [inline javascript/packages/orchestrator/README.md:391]:
```suggestion
❯ cd zombinet/javascript
❯ npm run zombie
```

**Review by wirednkod — CHANGES_REQUESTED** [review]

File: README.md [hunk 31]
```diff
@@ -321,17 +318,18 @@ Other examples are provided in the [examples](examples) directory.
...
 ```
 
 ### Download and install needed artifacts (optional)
 
 For an easier and faster setup of your local environment, run:
 
 ```bash
-node dist/cli.js setup <binaries>
+❯ cd zombinet/javascript/packages
+❯ node ./cli/dist/cli.js <binaries>
```
Comment (pepoviola) [inline README.md:332]:
```suggestion
❯ node ./cli/dist/cli.js setup <binaries>
```

File: README.md (hunk 9 above)
Comment (pepoviola) [reply README.md:354]: This path is old `/zombienet/dist:$PATH` is we keep this comment we should update it.

File: README.md [hunk 32]
```diff
@@ -343,27 +341,28 @@ You can use the following arguments:
...
 
 You can build it from source like this
 
 ```bash
-git clone git@github.com:paritytech/polkadot
-cd polkadot
-cargo build --profile testnet -p test-parachain-adder-collator
-export PATH=$(pwd)/target/testnet:$PATH
+❯ git clone git@github.com:paritytech/polkadot-sdk.git
+❯ cd polkadot-sdk/polkadot
```
Comment (pepoviola) [inline README.md:363]:
```suggestion
❯ cd polkadot-sdk
```
No need to `cd` into `polkadot` (and if you do the `export` below will not work).

File: docs/book/development.html (hunk 12 above)
Comment (pepoviola) [reply docs/book/development.html:164]: We shouldn't change the `html` files of the book, this is auto-generated right @wirednkod ?

File: javascript/README.md (hunk 23 above)
Comment (pepoviola) [reply javascript/README.md:328]: should be
```
npm run zombie -- setup <binaries>
```

File: javascript/packages/cli/README.md [hunk 33]
```diff
@@ -358,8 +357,8 @@ Please add the dir to your $PATH by running the command: export PATH=/home/<user
 You can build it from source like this
 
 ```bash
-git clone git@github.com:paritytech/polkadot
-cd polkadot
+❯ git clone git@github.com:paritytech/polkadot-sdk.git
+❯ cd polkadot-sdk/polkadot
```
Comment (pepoviola) [inline javascript/packages/cli/README.md:361]:
```suggestion
❯ cd polkadot-sdk
```

File: README.md (hunk 31 above)
Comment (wirednkod) [reply README.md:332]: For consistency reason I suggest:
```ts
❯ cd zombinet/javascript
❯ npm run zombie -- setup <binaries>
```

Comment (Neyromancer) [reply README.md:332]: @wirednkod, will stick with your suggestion then around these docs.

File: README.md [hunk 34]
```diff
@@ -321,17 +318,17 @@ Other examples are provided in the [examples](examples) directory.
...
 ```
 
 ### Download and install needed artifacts (optional)
 
 For an easier and faster setup of your local environment, run:
 
 ```bash
-node dist/cli.js setup <binaries>
+❯ cd zombinet/javascript
+❯ npm i && npm run zombie -- setup <binaries>
```
Comment (Neyromancer) [inline README.md:331]: command represented as recommended in the [comment](https://github.com/paritytech/zombienet/pull/1442#discussion_r1373801175)

File: README.md [hunk 35]
```diff
@@ -372,20 +370,26 @@ export PATH=$(pwd)/target/testnet:$PATH
...
-❯ node dist/cli.js
+❯ cd zombinet/javascript
+❯ npm run zombie
 
 Usage: zombienet [options] [command]
 
 Options:
-  -p, --provider <provider>                Override provider to use (choices: "podman",
-                                           "kubernetes", default: kubernetes)
+  -c, --spawn-concurrency <concurrency>    Number of concurrent spawning process to launch, default is 1
```
Comment (Neyromancer) [inline README.md:379]: make it aligned with current output, [see](https://github.com/paritytech/zombienet/blob/8f007c789ec12d204406db0200778da83bab2b4e/javascript/packages/cli/src/cli.ts#L76)

Comment (Neyromancer) [reply README.md:379]: here and others

File: README.md (hunk 9 above)
Comment (Neyromancer) [reply README.md:353]: returned previous path back

File: docs/book/development.html (hunk 12 above)
Comment (Neyromancer) [reply docs/book/development.html]: @pepoviola, I thought that they should be updated as they are stored at github as well

File: javascript/README.md [hunk 36]
```diff
@@ -119,7 +116,7 @@ Native provider doesn't run any extra layer/process at the moment.
 *For this example we will use the `macos` version of the executable*
 
 ```bash
-./zombienet-macos
+❯ ./zombienet-macos
 Usage: zombienet [options] [command]
```
Comment (wirednkod) [inline javascript/README.md:120]:
```suggestion
Usage: zombienet-macos [options] [command]
```

File: javascript/packages/orchestrator/README.md [hunk 37]
```diff
@@ -119,7 +116,7 @@ Native provider doesn't run any extra layer/process at the moment.
 *For this example we will use the `macos` version of the executable*
 
 ```bash
-./zombienet-macos
+❯ ./zombienet-macos
 Usage: zombienet [options] [command]
```
Comment (wirednkod) [inline javascript/packages/orchestrator/README.md:120]:
```suggestion
Usage: zombienet-macos [options] [command]
```

File: javascript/packages/orchestrator/README.md [hunk 38]
```diff
@@ -388,20 +386,26 @@ export PATH=$(pwd)/target/testnet:$PATH
 With the above steps completed, the `zombienet` CLI is ready to run:
 
 ```bash
-❯ node dist/cli.js
+❯ cd zombinet/javascript
+❯ npm run zombie
 
 Usage: zombienet [options] [command]
```
Comment (wirednkod) [inline javascript/packages/orchestrator/README.md:392]:
```suggestion
Usage: npm run zombie -- [options] [command]
```

**Review by wirednkod — APPROVED** [review]
lgtm. Beside some minor corrections. Thank you so much @Neyromancer . Thats a great amount of work on the doc!

**Comment by wirednkod** [comment]
@Neyromancer can you please sign the cla?

**Comment by Neyromancer** [comment]
@wirednkod, thank you for review and approve however i seem not posses the required rights to merge the PR. Could you merge it plz?

**Pull request merged**